lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -1030,11 +1030,7 @@ status_code_e amf_app_handle_pdu_session_accept(
                                &ue_context->amf_context._security);
   if (bytes > 0) {
     buffer->slen = bytes;
-    if (ue_context->ue_context_request) {
-      pdu_session_resource_setup_request(ue_context, ue_id, smf_ctx, buffer);
-    } else {
-      initial_context_setup_request(ue_id, &ue_context->amf_context, buffer);
-    }
+    pdu_session_resource_setup_request(ue_context, ue_id, smf_ctx, buffer);
```

Comment:
Can you add comment on why the check is no longer needed?

Reply:
ICS clubbed with session creation when UE Context Request IE not present.
In earlier behavior ICS was rejected if UE has not sent the “UE Context Request IE”.